<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="71" started-at="2019-09-28T16:20:41 IST" finished-at="2019-09-28T16:20:41 IST">
    <groups>
    </groups>
    <test name="Test" duration-ms="71" started-at="2019-09-28T16:20:41 IST" finished-at="2019-09-28T16:20:41 IST">
      <class name="config.HarvestOrder_Login">
        <test-method status="FAIL" signature="setUp()[pri:0, instance:config.HarvestOrder_Login@1d8d30f7]" name="setUp" is-config="true" duration-ms="1527" started-at="2019-09-28T16:20:39 IST" finished-at="2019-09-28T16:20:41 IST">
          <exception class="org.openqa.selenium.SessionNotCreatedException">
            <message>
              <![CDATA[Unable to create a new remote session. Please check the server log for more details. Original error: An unknown server-side error occurred while processing the command. Original error: Could not find 'adb.exe' in ["C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\platform-tools\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\emulator\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\tools\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\tools\\bin\\adb.exe"]. Do you have Android Build Tools installed at 'C:\Users\SuneelSirimisetti\AppData\Local\Android\Sdk\tools\bin\sdkmanager.bat'?
Build info: version: '4.0.0-alpha-1', revision: 'd1d3728cae', time: '2019-04-24T13:42:21'
System info: host: 'LAPTOP-R6T60MPJ', ip: '192.168.1.6', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_212'
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: Could not find 'adb.exe' in ["C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\platform-tools\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\emulator\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\tools\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\tools\\bin\\adb.exe"]. Do you have Android Build Tools installed at 'C:\Users\SuneelSirimisetti\AppData\Local\Android\Sdk\tools\bin\sdkmanager.bat'?
    at getResponseForW3CError (C:\Program Files\Appium\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:826:9)
    at asyncHandler (C:\Program Files\Appium\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:447:37)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Build info: version: '4.0.0-alpha-1', revision: 'd1d3728cae', time: '2019-04-24T13:42:21'
System info: host: 'LAPTOP-R6T60MPJ', ip: '192.168.1.6', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_212'
Driver info: driver.version: AndroidDriver]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: An unknown server-side error occurred while processing the command. Original error: Could not find 'adb.exe' in ["C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\platform-tools\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\emulator\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\tools\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\tools\\bin\\adb.exe"]. Do you have Android Build Tools installed at 'C:\Users\SuneelSirimisetti\AppData\Local\Android\Sdk\tools\bin\sdkmanager.bat'?
Build info: version: '4.0.0-alpha-1', revision: 'd1d3728cae', time: '2019-04-24T13:42:21'
System info: host: 'LAPTOP-R6T60MPJ', ip: '192.168.1.6', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_212'
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: Could not find 'adb.exe' in ["C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\platform-tools\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\emulator\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\tools\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\tools\\bin\\adb.exe"]. Do you have Android Build Tools installed at 'C:\Users\SuneelSirimisetti\AppData\Local\Android\Sdk\tools\bin\sdkmanager.bat'?
    at getResponseForW3CError (C:\Program Files\Appium\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:826:9)
    at asyncHandler (C:\Program Files\Appium\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:447:37)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Build info: version: '4.0.0-alpha-1', revision: 'd1d3728cae', time: '2019-04-24T13:42:21'
System info: host: 'LAPTOP-R6T60MPJ', ip: '192.168.1.6', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_212'
Driver info: driver.version: AndroidDriver
at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:208)
at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:217)
at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:239)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:577)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:211)
at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:323)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
at io.appium.java_client.DefaultGenericMobileDriver.<init>(DefaultGenericMobileDriver.java:37)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:86)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:96)
at io.appium.java_client.android.AndroidDriver.<init>(AndroidDriver.java:94)
at config.HarvestOrder_Login.setUp(HarvestOrder_Login.java:38)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:340)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:294)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:345)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.reflect.InvocationTargetException
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:186)
... 30 more
Caused by: org.openqa.selenium.WebDriverException: An unknown server-side error occurred while processing the command. Original error: Could not find 'adb.exe' in ["C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\platform-tools\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\emulator\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\tools\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\tools\\bin\\adb.exe"]. Do you have Android Build Tools installed at 'C:\Users\SuneelSirimisetti\AppData\Local\Android\Sdk\tools\bin\sdkmanager.bat'?
Build info: version: '4.0.0-alpha-1', revision: 'd1d3728cae', time: '2019-04-24T13:42:21'
System info: host: 'LAPTOP-R6T60MPJ', ip: '192.168.1.6', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_212'
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: Could not find 'adb.exe' in ["C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\platform-tools\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\emulator\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\tools\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\tools\\bin\\adb.exe"]. Do you have Android Build Tools installed at 'C:\Users\SuneelSirimisetti\AppData\Local\Android\Sdk\tools\bin\sdkmanager.bat'?
    at getResponseForW3CError (C:\Program Files\Appium\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:826:9)
    at asyncHandler (C:\Program Files\Appium\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:447:37)
    at process._tickCallback (internal/process/next_tick.js:68:7)
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.lang.reflect.Constructor.newInstance(Unknown Source)
at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
at java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)
at java.util.Spliterators$ArraySpliterator.tryAdvance(Unknown Source)
at java.util.stream.ReferencePipeline.forEachWithCancel(Unknown Source)
at java.util.stream.AbstractPipeline.copyIntoWithCancel(Unknown Source)
at java.util.stream.AbstractPipeline.copyInto(Unknown Source)
at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
at java.util.stream.FindOps$FindOp.evaluateSequential(Unknown Source)
at java.util.stream.AbstractPipeline.evaluate(Unknown Source)
at java.util.stream.ReferencePipeline.findFirst(Unknown Source)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
... 35 more
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.SessionNotCreatedException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="Sum()[pri:0, instance:config.HarvestOrder_Login@1d8d30f7]" name="Sum" duration-ms="0" started-at="2019-09-28T16:20:41 IST" finished-at="2019-09-28T16:20:41 IST">
          <exception class="org.openqa.selenium.SessionNotCreatedException">
            <message>
              <![CDATA[Unable to create a new remote session. Please check the server log for more details. Original error: An unknown server-side error occurred while processing the command. Original error: Could not find 'adb.exe' in ["C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\platform-tools\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\emulator\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\tools\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\tools\\bin\\adb.exe"]. Do you have Android Build Tools installed at 'C:\Users\SuneelSirimisetti\AppData\Local\Android\Sdk\tools\bin\sdkmanager.bat'?
Build info: version: '4.0.0-alpha-1', revision: 'd1d3728cae', time: '2019-04-24T13:42:21'
System info: host: 'LAPTOP-R6T60MPJ', ip: '192.168.1.6', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_212'
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: Could not find 'adb.exe' in ["C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\platform-tools\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\emulator\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\tools\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\tools\\bin\\adb.exe"]. Do you have Android Build Tools installed at 'C:\Users\SuneelSirimisetti\AppData\Local\Android\Sdk\tools\bin\sdkmanager.bat'?
    at getResponseForW3CError (C:\Program Files\Appium\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:826:9)
    at asyncHandler (C:\Program Files\Appium\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:447:37)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Build info: version: '4.0.0-alpha-1', revision: 'd1d3728cae', time: '2019-04-24T13:42:21'
System info: host: 'LAPTOP-R6T60MPJ', ip: '192.168.1.6', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_212'
Driver info: driver.version: AndroidDriver]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: An unknown server-side error occurred while processing the command. Original error: Could not find 'adb.exe' in ["C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\platform-tools\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\emulator\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\tools\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\tools\\bin\\adb.exe"]. Do you have Android Build Tools installed at 'C:\Users\SuneelSirimisetti\AppData\Local\Android\Sdk\tools\bin\sdkmanager.bat'?
Build info: version: '4.0.0-alpha-1', revision: 'd1d3728cae', time: '2019-04-24T13:42:21'
System info: host: 'LAPTOP-R6T60MPJ', ip: '192.168.1.6', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_212'
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: Could not find 'adb.exe' in ["C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\platform-tools\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\emulator\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\tools\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\tools\\bin\\adb.exe"]. Do you have Android Build Tools installed at 'C:\Users\SuneelSirimisetti\AppData\Local\Android\Sdk\tools\bin\sdkmanager.bat'?
    at getResponseForW3CError (C:\Program Files\Appium\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:826:9)
    at asyncHandler (C:\Program Files\Appium\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:447:37)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Build info: version: '4.0.0-alpha-1', revision: 'd1d3728cae', time: '2019-04-24T13:42:21'
System info: host: 'LAPTOP-R6T60MPJ', ip: '192.168.1.6', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_212'
Driver info: driver.version: AndroidDriver
at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:208)
at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:217)
at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:239)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:577)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:211)
at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:323)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
at io.appium.java_client.DefaultGenericMobileDriver.<init>(DefaultGenericMobileDriver.java:37)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:86)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:96)
at io.appium.java_client.android.AndroidDriver.<init>(AndroidDriver.java:94)
at config.HarvestOrder_Login.setUp(HarvestOrder_Login.java:38)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:340)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:294)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:345)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.reflect.InvocationTargetException
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:186)
... 30 more
Caused by: org.openqa.selenium.WebDriverException: An unknown server-side error occurred while processing the command. Original error: Could not find 'adb.exe' in ["C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\platform-tools\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\emulator\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\tools\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\tools\\bin\\adb.exe"]. Do you have Android Build Tools installed at 'C:\Users\SuneelSirimisetti\AppData\Local\Android\Sdk\tools\bin\sdkmanager.bat'?
Build info: version: '4.0.0-alpha-1', revision: 'd1d3728cae', time: '2019-04-24T13:42:21'
System info: host: 'LAPTOP-R6T60MPJ', ip: '192.168.1.6', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_212'
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: Could not find 'adb.exe' in ["C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\platform-tools\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\emulator\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\tools\\adb.exe","C:\\Users\\SuneelSirimisetti\\AppData\\Local\\Android\\Sdk\\tools\\bin\\sdkmanager.bat\\tools\\bin\\adb.exe"]. Do you have Android Build Tools installed at 'C:\Users\SuneelSirimisetti\AppData\Local\Android\Sdk\tools\bin\sdkmanager.bat'?
    at getResponseForW3CError (C:\Program Files\Appium\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:826:9)
    at asyncHandler (C:\Program Files\Appium\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:447:37)
    at process._tickCallback (internal/process/next_tick.js:68:7)
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.lang.reflect.Constructor.newInstance(Unknown Source)
at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
at java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)
at java.util.Spliterators$ArraySpliterator.tryAdvance(Unknown Source)
at java.util.stream.ReferencePipeline.forEachWithCancel(Unknown Source)
at java.util.stream.AbstractPipeline.copyIntoWithCancel(Unknown Source)
at java.util.stream.AbstractPipeline.copyInto(Unknown Source)
at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
at java.util.stream.FindOps$FindOp.evaluateSequential(Unknown Source)
at java.util.stream.AbstractPipeline.evaluate(Unknown Source)
at java.util.stream.ReferencePipeline.findFirst(Unknown Source)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
... 35 more
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.SessionNotCreatedException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Sum -->
        <test-method status="SKIP" signature="End()[pri:0, instance:config.HarvestOrder_Login@1d8d30f7]" name="End" is-config="true" duration-ms="-1569667841242" started-at="2019-09-28T16:20:41 IST" finished-at="1970-01-01T05:30:00 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- End -->
      </class> <!-- config.HarvestOrder_Login -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
